proxy-robots-module/src/main/java/orwell/proxy/zmq/ZmqMessageBroker.java

```diff
@@ -120,8 +122,17 @@ public boolean isConnectedToServer() {
         return isConnected;
     }
 
-    public int getNbMessagesSkipped() {
-        return nbMessagesSkipped;
+    public int getNbSuccessiveMessagesSkipped() {
+        return nbSuccessiveSameMessages;
+    }
+
+    public int getNbBadMessages() {
+        return nbBadMessages;
+    }
+
+    private boolean hasSocketRecvTimeout() {
```

Comment:
I am a bit uncomfortable with the name of the method. Maybe hasReceiveTimeoutExpired ?